CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Please join the mailing list and our bi-weekly calls! See details in next sectio
 
```

Comment:
also inclined to remove this section and incorporate to a lighter degree in readme, see #981